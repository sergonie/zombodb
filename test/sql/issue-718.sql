-- should return a count of zero
select count(*) from so_users where so_users ==> 'about_me:foo.bar1.com';