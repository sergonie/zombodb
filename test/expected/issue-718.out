-- should return a count of zero
select count(*) from so_users where so_users ==> 'about_me:foo.bar1.com';
 count 
-------
     0
(1 row)

